CHANGELOG.md

```diff
@@ -7,9 +7,14 @@ All notable changes to this project will be documented in this file.
 
 #### Added
 
-- Added support for Amazon Linux 2022 in Vulnerability Detector. [#13034](https://github.com/wazuh/wazuh/issue/13034)
 - wazuh-authd can now generate X509 certificates. [#13559](https://github.com/wazuh/wazuh/pull/13559)
-- Introduced a new CLI to manage features related to the Wazuh API RBAC resources. ([#13797](https://github.com/wazuh/wazuh/pull/13797))
+- Introduced a new CLI to manage features related to the Wazuh API RBAC resources. [#13797](https://github.com/wazuh/wazuh/pull/13797)
+- Added support for Amazon Linux 2022 in Vulnerability Detector. [#13034](https://github.com/wazuh/wazuh/issue/13034)
+- Added support for Alma Linux in Vulnerability Detector. [#16343](https://github.com/wazuh/wazuh/pull/16343)
+- Added support for Debian 12 in Vulnerability Detector. [#18542](https://github.com/wazuh/wazuh/pull/18542)
+- Added mechanism in wazuh-db to identify fragmentation and perform vacuum. [#14953](https://github.com/wazuh/wazuh/pull/14953)
+- Added an option to set whether the manager should ban newer agents. [#18333](https://github.com/wazuh/wazuh/pull/18333)
+- Added mechanism to prevent wazuh agents connections to lower manager versions. [#15661](https://github.com/wazuh/wazuh/pull/15661)
```

Comment:
So far, the PR number is always in parentheses in all previous changelogs. Check this out in all new entries:

```suggestion
- wazuh-authd can now generate X509 certificates. ([#13559](https://github.com/wazuh/wazuh/pull/13559))
- Introduced a new CLI to manage features related to the Wazuh API RBAC resources. ([#13797](https://github.com/wazuh/wazuh/pull/13797))
- Added support for Amazon Linux 2022 in Vulnerability Detector. ([#13034](https://github.com/wazuh/wazuh/issue/13034))
- Added support for Alma Linux in Vulnerability Detector. ([#16343](https://github.com/wazuh/wazuh/pull/16343))
- Added support for Debian 12 in Vulnerability Detector. ([#18542](https://github.com/wazuh/wazuh/pull/18542))
- Added mechanism in wazuh-db to identify fragmentation and perform vacuum. ([#14953](https://github.com/wazuh/wazuh/pull/14953))
- Added an option to set whether the manager should ban newer agents. ([#18333](https://github.com/wazuh/wazuh/pull/18333))
- Added mechanism to prevent wazuh agents connections to lower manager versions. ([#15661](https://github.com/wazuh/wazuh/pull/15661))
```